notebooks/en/rag_evaluation.ipynb

```diff
@@ -37,47 +37,76 @@
   },
```

Comment:
<p>Can you please remove the output of this cell? We don't need it, and it doesn't render well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>